easybuild/tools/module_generator.py

```diff
@@ -221,46 +221,87 @@ def _modulerc_check_module_version(self, module_version):
 
         return res
 
-    def modulerc(self, module_version=None):
+    def _write_modulerc_file(self, modulerc_path, modulerc_txt, wrapped_mod_name=None):
         """
-        Generate contents of .modulerc file, in Tcl syntax (compatible with all module tools, incl. Lmod)
+        Write modulerc file with specified contents.
+
+        :param modulerc_path: location of .modulerc file to write
+        :param modulerc_txt: contents of .modulerc file
+        :param wrapped_mod_name: name of module file for which a wrapper is defined in the .modulerc file (if any)
+        """
+        if os.path.exists(modulerc_path) and not build_option('force'):
```

Comment:
Should we also allow overwriting the `.modulerc` in the case the user passes `--rebuild`?

Reply:
Good point... Fixed in e1161b525!